### PR TITLE
Epic/interactive pathways

### PR DESCRIPTION
**Pull request by glassdimly**
## Jira

https://app.asana.com/0/1126340469288208/1129426257662815/f

## Summary

Fix the nav line strikethrough so it doesn't go through the last nav item, fixing spacing on last item.

## How to test

Compare against the comps and review in light of the ticket.

### CODE REVIEW

**Comment by sghoweri** [comment]
Hey @joekarasek @glassdimly one quick comment on some of the prop names I’m seeing here: what’s the thought process behind having namespaces on the prop names themselves?

I think it should be pretty clear that an `animation-name` prop on the `<bolt-animation>` component is a specific prop name to this component (vs the current `bolt-animation-name` prop I’m seeing). ^ this also keeps the naming convention across existing and future components more consistent and concise.

TLDR: There shouldn’t be a need to prefix Bolt-specific prop names or prop values on Bolt web components! 

PS. Namespacing custom prop values that can get added by a consumer of Bolt would probably be fair game. 

CC @mikemai2awesome @danielamorse

**Comment by margoromo** [comment]
Added http://vjira2:8080/browse/BDS-1714 and http://vjira2:8080/browse/BDS-1715 for @sghoweri and @mikemai2awesome to review.

**Comment by EvanLovely** [comment]
@sghoweri Didn't notice your comment above till now; sorry!

> Hey @joekarasek @glassdimly one quick comment on some of the prop names I’m seeing here: what’s the thought process behind having namespaces on the prop names themselves?
> I think it should be pretty clear that an animation-name prop on the <bolt-animation> component is a specific prop name to this component (vs the current bolt-animation-name prop I’m seeing). ^ this also keeps the naming convention across existing and future components more consistent and concise.

This is all taken care of now, the component has build in and build out animations that are used like this:

```html
<bolt-animate in="fade-in" in-duration="750" out="fade-out-slide-down">
  anything goes here
</bolt-animate>
```

Full details in it's schema file: https://github.com/bolt-design-system/bolt/blob/epic/interactive-pathways/packages/components/bolt-animate/animate.schema.js

File: docs-site/src/pages/pattern-lab/_patterns/04-pages/80-micro-journeys/10-character.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<style>
+  .u-demo-padding {
+    padding: 100px;
+  }
+</style>
+<div class="u-demo-padding">
+  <bolt-character
+      characterUrl="https://github.com/basaltinc/temp-pega-dummy-assets/raw/master/animated-bands-ltr.png"
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/04-pages/80-micro-journeys/10-character.twig]: Web component props need to use the `kebab-case` convention.

**Comment by EvanLovely** [comment]
@sghoweri @mikemai2awesome OK, we've got a bit of a GitHub PR tangle going on; please bear with us while we do some housekeeping on all this! Here's what's up:

- This is not the main Epic branch for Interactive Pathways (aka Micro Journeys), that branch is `epic/interactive-pathways` and the Draft PR is #1207 
- This is a `fix` PR for that branch; however the git merge direction is backwards and you can't flip that in the GitHub UI; so I made a new PR that supersedes this one: #1316 

I'm going to close this PR out now; but I'll be capturing your feedback for discussion in the main PR #1207 or addressing it directly in-line; thanks for the thoughts!

**Comment by EvanLovely** [comment]
@mikemai2awesome The component names, package names, file and folder locations are in the middle of a refactor and cleanup currently. We are on the same page; these are the final NPM packages that will be added by us:

- `@bolt/animations` Visual Styles for animations
- `@bolt/animate` Web Component that animates children
- `@bolt/editor` Editor template (it will have Twig template) and JS app
- `@bolt/micro-journeys` Collection of multiple Web Components that are highly animatable

**Comment by EvanLovely** [comment]
@mikemai2awesome So this code enables some awesome stuff with the editor; it makes the dropzones animatable - makes for a better user experience when building these UI via GUI instead of code.

```html
<bolt-character
  characterUrl="https://github.com/basaltinc/temp-pega-dummy-assets/raw/master/animated-bands-ltr.png"
>
  <bolt-animate slot="top" idle="pulse">
  </bolt-animate>
  <bolt-animate slot="left" idle="pulse">
  </bolt-animate>
  <bolt-animate slot="right" idle="pulse">
  </bolt-animate>
  <bolt-animate slot="bottom" idle="pulse">
  </bolt-animate>
  <bolt-icon slot="icon" idle="none">
  </bolt-icon>
</bolt-character>
```

File: docs-site/src/pages/pattern-lab/_patterns/04-pages/80-micro-journeys/_editor-wrapper.twig [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<style>
```
[the remark this answers is not in the document]
Comment (EvanLovely) [reply docs-site/src/pages/pattern-lab/_patterns/04-pages/80-micro-journeys/_editor-wrapper.twig]: So this and all the other code comments are on heavy WIP areas; we'll get it all in a good spot before merge.
